composer.json

```
@@ -6,20 +6,19 @@
     "minimum-stability": "dev",
     "prefer-stable": true,
     "require": {
-        "drupal/core": "^8.7",
-        "php": ">=7.2"
+        "drupal/core": "^8.9 || ^9.1",
+        "php": ">=7.3"
     },
     "require-dev": {
-        "composer/installers": "~1.5",
+        "composer/installers": "^1.11",
+        "drupal/core-composer-scaffold": "^8.9 || ^9.1",
         "drupal/config_devel": "~1.2",
-        "drupal/core-composer-scaffold": "^8.8",
         "drupal/drupal-extension": "~4.0",
-        "drush/drush": "~9.0@stable",
-        "guzzlehttp/guzzle": "~6.3",
-        "openeuropa/code-review": "~1.0.0-beta2",
-        "openeuropa/drupal-core-require-dev": "^8.7",
-        "openeuropa/task-runner": "~1.0.0-beta5",
-        "phpunit/phpunit": "~6.0"
+        "drush/drush": "^10.3",
```

We can keep `~` for this constraint as was before.
```suggestion
        "drush/drush": "^10.3",
```